Resources/public/js/alloyeditor/buttons/linkedit.js

```diff
@@ -312,23 +312,15 @@ YUI.add('ez-alloyeditor-button-linkedit', function (Y) {
             var editor = this.props.editor.get('nativeEditor'),
                 linkUtils = new CKEDITOR.Link(editor),
                 selection = editor.getSelection(),
-                bookmarks = selection.createBookmarks(),
-                scrollY;
+                bookmarks = selection.createBookmarks();
 
             linkUtils.remove(this.state.element, {advance: true});
 
             selection.selectBookmarks(bookmarks);
 
             this.props.cancelExclusive();
 
-            if (navigator.userAgent.indexOf('Chrome') > -1) {
```

Comment:
Why?

Reply:
It looks like I've deleted it by accident. Thanks for vigilance